lib/trino-hive-formats/src/main/java/io/trino/hive/formats/ion/IonDecoderFactory.java

```diff
@@ -71,7 +72,7 @@ public static IonDecoder buildDecoder(List<Column> columns)
     {
         return RowDecoder.forFields(
                 columns.stream()
-                        .map(c -> new RowType.Field(Optional.of(c.name()), c.type()))
+                        .map(c -> new RowType.Field(Optional.of(c.name().toLowerCase(Locale.ROOT)), c.type()))
```

Comment:
Is this necessary? AFAICT the lowercasing in `RowDecoder.forFields()` is sufficient.

Reply:
Thats right, `RowDecoder.forFields` already performs lower casing. Will remove it from here.